docs/about/release-notes.rst

```diff
@@ -3,29 +3,48 @@
 Release Notes
 =============
 
-Since v0.4
+Since v0.5
 ----------
 
 Features
 ~~~~~~~~
 
-* New ``profiler`` plotting functionality where one of the slider dimensions can be displayed as a profile in a subplot (1D and 2D projections only).
+Breaking changes
+~~~~~~~~~~~~~~~~
+
+Contributors
+~~~~~~~~~~~~
+
+v0.5 (January 2021)
+-------------------
+
+Features
+~~~~~~~~
+
+* New ``profile`` plotting functionality where one of the slider dimensions can be displayed as a profile in a subplot (1-D and 2-D projections only).
 * Sliders have a thickness slider associated with them and can be used to show slices of arbitrary thickness.
 * Can hide/show individual masks on plots.
 * Can toggle log scale of axes and colorbar with buttons in figure toolbar.
 * Add binned data support, replacing "event list" dtypes as well as "realign" support.
+* Plotting of event data (binned data) with dynamic resampling with "infinite zoom" functionality.
 * Value-based slicing support.
-* Support for saving scipp objects to HDF5.
 * Possibility to plot Scipp objects using ``my_data_array.plot()`` in addition to the classical ``plot()`` free function.
+* Support for saving and loading scipp data structures to HDF5.
```

Comment:
https://scipp.github.io/user-guide/reading-and-writing-files.html?highlight=hdf5

Reply:
See below.